cloud-data-protection/src/common/formatting/fileFormat.ts

```diff
@@ -0,0 +1,10 @@
+export const formatBytes = (bytes: number) => {
+    if (bytes === 0) return '0 Bytes';
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/olivierbouchomsfontys/cloud-data-protection/pullRequest?prid=7431242)
```suggestion
    if (bytes === 0) {return '0 Bytes';}
```